spec.bs

```diff
@@ -747,16 +747,21 @@ To <dfn>build bid generators map</dfn> given an [=auction config=] |auctionConfi
 To <dfn>generate a bid</dfn> given an [=ordered map=] |allTrustedBiddingSignals|, a 
 [=string=] |auctionSignals|, an [=ordered map=] whose [=map/keys=] are [=strings=] and
 whose [=map/values=] are {{any}} |browserSignals|, a [=string=] |perBuyerSignals|, a
-[=duration=] |perBuyerTimeout| in milliseconds, and an [=interest group=] |ig|:
+[=duration=] |perBuyerTimeout| in milliseconds, an [=interest group=] |ig|, and a [=moment=]
+|auctionStartTime|:
   1. TODO: Let |interestGroup| be ... from |ig| ... minus priority and prioritySignalsOverrides and any browser-defined pieces
   1. [=map/Set=] |browserSignals|["`joinCount`"] to the sum of |ig|'s
      [=interest group/join counts=] for all days within the last 30 days.
   1. [=map/Set=] |browserSignals|["`bidCount`"] to the sum of |ig|'s
      [=interest group/bid counts=] for all days within the last 30 days.
-  1. [=map/Set=] |browserSignals|["`prevWins`"] to a [=list=] containing [=tuples=] of the
-     time and the corresponding winning [=interest group ad=] from |ig|'s
-     [=interest group/previous wins=] field with a date within the last 30 days. The time
-     field is specified in seconds relative to the start of the auction.
+  1. Let |prevWins| be a [=list=] of [=tuples=], with each [=tuple=] consists of a time (a 
+    [=duration=] in milliseconds), and adJson (a [=string=]).

```

Comment:
Down below this is defined as `ad json` - we should make it consistent and link to it. Note that in https://wicg.github.io/turtledove/#convert-generatebidoutput-to-generated-bid there is a similarly named `adJSON`. Based on https://www.w3.org/TR/design-principles/#casing-rules, that's probably the spelling we want here.